src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -1483,6 +1487,9 @@ public void ReceiveFromV4BoundToSpecificV6_NotReceived()
             });
         }
 
+        // Binds to a specific port on 'connectTo' which on Unix may already be in use
+        // Test preserved if we change these tests to run in series (instead of parallel) to avoid port collision
+#if false
```

Comment:
We usually don't ifdef away code like this and leave it as "dead code".  At a minimum, there needs to be a tracking issue associated with something like this.  What value is there is ifdef'ing this code away?